android/app-example/src/main/java/com/badoo/ribs/example/rib/dialog_example/dialog/SimpleDialog.kt

```diff
@@ -0,0 +1,24 @@
+package com.badoo.ribs.example.rib.dialog_example.dialog
+
+import com.badoo.ribs.dialog.Dialog
+import com.badoo.ribs.dialog.Dialog.CancellationPolicy.Cancellable
+import com.badoo.ribs.dialog.Dialog.Event.Negative
+import com.badoo.ribs.dialog.Dialog.Event.Neutral
+import com.badoo.ribs.dialog.Dialog.Event.Positive
+
+
+class SimpleDialog : Dialog<Dialog.Event>({
+    // Inject (Smart)Resources in constructor to resolve texts if needed
+    title = "Simple dialog"
+    message = "Lorem ipsum dolor sit amet"
+    buttons {
+        positive("Yay!", Positive)
+        negative("No way", Negative)
+        neutral("Meh?", Neutral)
+    }
+
+    cancellationPolicy = Cancellable(
```

Comment:
factory method as above?

Reply:
Not sure we can do that, as if you have Cancellable, you have to have the event defined. Suggestions?

Reply:
Right, makes sense as it is